demo-drone-app/src/components/LandingPage.vue

```diff
@@ -87,17 +87,35 @@
         class="my-5"
         align-center
       >
-        <v-flex xs12 sm4 class="my-3">
-          <div class="text-xs-center">
-            <h2 class="headline" style="margin-bottom:5px;">Be ahead of the curve.</h2>
-            <span class="subheading">
-              Have a drone policy that fuels innovation.
-            </span>
-          </div>
-        </v-flex>
         <v-flex xs12>
           <v-container grid-list-xl>
             <v-layout row wrap align-center>
+              <v-flex xs12 md4>
```

Comment:
Maybe put style="height:100vh" so that the div takes up the entire page? I think it'd look cleaner.

Reply:
also you don't need md4 if it is only visible at xs